package.json

```diff
@@ -3,61 +3,51 @@
   "version": "1.7.17",
   "private": false,
   "license": "MIT",
+  "type": "module",
   "scripts": {
     "test": "jest",
     "test-watch": "jest --watch",
     "test-coverage": "jest --coverage",
-    "serve": "serve public",
-    "start": "npx mix watch",
-    "build": "npx mix --production"
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint .",
+    "preview": "vite preview"
   },
   "devDependencies": {
-    "@babel/preset-react": "^7.22.3",
-    "@babel/preset-typescript": "^7.21.5",
-    "@emotion/babel-plugin": "^11.11.0",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^14.0.0",
-    "@types/jest": "^29.5.1",
-    "@types/luxon": "^3.3.0",
-    "@types/react": "^18.2.7",
-    "@types/react-dom": "^18.2.4",
-    "@types/react-infinite-scroller": "^1.2.3",
-    "@types/react-test-renderer": "^18.0.0",
-    "@typescript-eslint/eslint-plugin": "^6.19.0",
-    "@typescript-eslint/parser": "^6.19.0",
-    "eslint": "^8.56.0",
-    "eslint-config-standard-with-typescript": "^43.0.0",
-    "eslint-import-resolver-typescript": "^3.6.1",
-    "eslint-plugin-import": "^2.29.1",
-    "eslint-plugin-n": "^16.6.2",
-    "eslint-plugin-promise": "^6.1.1",
-    "eslint-plugin-react": "^7.33.2",
-    "jest": "^29.5.0",
-    "jest-environment-jsdom": "^29.6.1",
-    "laravel-mix": "^6.0.49",
-    "react-test-renderer": "^18.2.0",
-    "serve": "^14.2.0",
-    "ts-loader": "^9.4.3",
-    "typescript": "^5.3.3"
+    "@eslint/js": "^9.11.1",
+    "@testing-library/jest-dom": "^6.5.0",
+    "@testing-library/react": "^16.0.1",
+    "@types/jest": "^29.5.13",
+    "@types/luxon": "^3.4.2",
+    "@types/react": "^18.3.10",
+    "@types/react-dom": "^18.3.0",
+    "@types/react-infinite-scroller": "^1.2.5",
+    "@vitejs/plugin-react": "^4.3.2",
+    "eslint": "^9.11.1",
+    "eslint-plugin-react-hooks": "^5.1.0-rc.0",
+    "eslint-plugin-react-refresh": "^0.4.12",
+    "globals": "^15.9.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
+    "ts-jest": "^29.2.5",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.5.3",
+    "typescript-eslint": "^8.7.0",
+    "vite": "^5.4.8",
+    "vite-plugin-css-injected-by-js": "^3.5.2"
   },
   "dependencies": {
-    "@code4recovery/spec": "1.0.9",
-    "@emotion/react": "^11.11.1",
+    "@code4recovery/spec": "^1.1.4",
+    "@emotion/react": "^11.13.3",
     "deepmerge": "^4.3.1",
-    "luxon": "^3.3.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "luxon": "^3.5.0",
+    "mapbox-gl": "^3.7.0",
```

Comment:
it complained that this was missing - we were referencing files in it - so now it's an explicit dependency